requirements.txt

```diff
@@ -1,4 +1,4 @@
-jsonschema
+jsonschema==3.2.0
```

Comment:
Maybe remove that dependency completely as is being required already by the `openapi-schema-validator` dependency (and also remove from `setup.cfg`)?

Reply:
That's true but we use `jsonschema` directly also and version requirements for these two libraries can be different.

Reply:
But pinning (`==`) on one side of two never solves such a different needs problem with pip - there must be intervals of versions so thatpip can find a non-empty intersect of versions (as only a single version can be installed inside an environment.

Reply:
I agree, this should be caret requirement `^3.2.0`

Reply:
Yeah, I plan to move the library to poetry, that's why